ossapi/ossapiv2.py

```diff
@@ -449,6 +449,25 @@ def beatmap(self, beatmap_id: int) -> Beatmap:
         """
         return self._get(Beatmap, f"/beatmaps/{beatmap_id}")
 
+    # /beatmapsets
+    # ____________
+
+    @request
+    def beatmapset_discussion_posts(self,
+        beatmapset_session_id: int = None,
+        limit: int = None,
+        page: int = None,
+        sort: SortOrder = None,
```

Comment:
this should be `SortOrderT` with a new sort order type at the top of the file, see existing enums for examples